src/osd/scrubber/pg_scrubber.cc

```diff
@@ -488,35 +496,39 @@ void PgScrubber::rm_from_osd_scrubbing()
   }
 }
 
-void PgScrubber::on_pg_activate(const requested_scrub_t& request_flags)
+/*
+ * Note: referring to m_planned_scrub here is temporary, as this set of
+ * scheduling flags will be removed in a followup PR.
+ */
+void PgScrubber::on_PrimaryActive()
```

Comment:
Please don't combine snake_case and CamelCase.  Also, I'd prefer that PGScrubber methods express what they actually do rather than when they are called.  This method registers the PG with the scheduler, so I think it should be PgScrubber::schedule_scrub_with_osd() or similar.